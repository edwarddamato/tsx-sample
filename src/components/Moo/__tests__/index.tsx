import * as React from 'react';
import { Moo } from '../';
import { IUser } from '../../../api/github/interfaces.d';
import { create } from 'react-test-renderer';

it('renders correctly', () => {
  const user: IUser = {
    Name: "Edward D'Amato",
    Location: 'London',
    Bio: 'Moo'
  };
  const moo = create(<Moo user={user} />).toJSON();
  expect(moo).toMatchSnapshot();
});
