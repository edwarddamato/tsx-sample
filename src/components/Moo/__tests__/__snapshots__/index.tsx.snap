// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="component_moo"
>
  <div>
    Name: 
    Edward D'Amato
  </div>
  <div>
    Location: 
    London
  </div>
  <div>
    Bio: 
    Moo
  </div>
</div>
`;
